src/Forms/TextDisplay.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement, FunctionComponent } from 'react';
+import styled from 'styled-components';
+import { fromTheme } from '../Theme';
+
+// An enum that represents the possible values for the label's positioning
+export enum POSITION {
+  TOP = 'top',
+  LEFT = 'left',
+}
+
+export interface TextDisplayProps {
+  /** The id of the label tied to this text display field */
+  id: string;
+  /** Specifies the label text */
+  label: string;
+  /** The name of the text display field */
+  name: string;
+  /** The current value in the text display field */
+  value: string;
+  /** Allows you to pass in a label position property from the POSITION enum
+   * @default LEFT
+   */
+  labelPosition?: POSITION;
+  /** If true, label will be visible
+   * @default true
+   */
+  isLabelVisible?: boolean;
+}
+
+export interface StyledDisplayLabelProps {
+  /** Allows you to pass in a label position property from the POSITION enum
+   * @default LEFT
+   */
+  labelPosition?: POSITION;
+}
+
+export interface StyledTextDisplayProps {
```

Comment:
Is `StyledTextDisplayProps` being used anywhere? This seems like it can be safely removed.